version.cmake

```diff
@@ -4,8 +4,8 @@
 set ( HPCC_NAME "Community Edition" )
 set ( HPCC_PROJECT "community" )
 set ( HPCC_MAJOR 8 )
-set ( HPCC_MINOR 2 )
+set ( HPCC_MINOR 1 )
 set ( HPCC_POINT 0 )
-set ( HPCC_MATURITY "rc" )
-set ( HPCC_SEQUENCE 1 )
+set ( HPCC_MATURITY "trunk" )
+set ( HPCC_SEQUENCE 0 )
```

Comment:
Looks like version.cmake changes accidentally released, needs rolling back

Reply:
I make it the same as candidate-8.2.x:
set ( HPCC_NAME "Community Edition" )
set ( HPCC_PROJECT "community" )
set ( HPCC_MAJOR 8 )
set ( HPCC_MINOR 2 )
set ( HPCC_POINT 1 )
set ( HPCC_MATURITY "closedown" )
set ( HPCC_SEQUENCE 0 )

Reply:
@wangkx - versiion.cmake should not be in the PR at all.
Maybe you need to rebase.